src/components/editor/Bindings/Tabs.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Button, Tab, Tabs } from '@mui/material';
+import { MuiTabProps } from 'components/editor/Bindings/types';
+import { Dispatch, SetStateAction, useMemo } from 'react';
+import { useIntl } from 'react-intl';
+
+interface BindingTabsProps {
+    selectedTab: number;
+    setSelectedTab: Dispatch<SetStateAction<number>>;
+}
+
+export const tabProps: MuiTabProps[] = [
+    {
+        label: 'workflows.collectionSelector.tab.resourceConfig',
+        value: 'config',
+    },
+    {
+        label: 'workflows.collectionSelector.tab.collectionSchema',
+        value: 'schema',
+    },
+];
+
+function BindingsTabs({ selectedTab, setSelectedTab }: BindingTabsProps) {
+    const intl = useIntl();
+
+    const tabs = useMemo(
+        () =>
+            tabProps.map((tabProp, index) => (
+                <Tab
+                    key={`collection-selection-tabs-${tabProp.label}`}
+                    label={intl.formatMessage({
+                        id: tabProp.label,
+                    })}
+                    component={Button}
```

Comment:
This should be a `Link` so the tabs looks like Admin's 
![image](https://user-images.githubusercontent.com/270078/205685261-a61a968f-768a-4abd-ade1-0734d3351430.png)